docs/admin/logging/awslogs.md

```diff
@@ -63,6 +63,14 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+# awslogs-stream-template
+
+Configure a [log stream](http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/WhatIsCloudWatchLogs.html)
```

Comment:
You might try something like this:

Specify a template to automatically generate the [log stream](http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/WhatIsCloudWatchLogs.html) name to use. The template supports [logger.Context](https://godoc.org/github.com/docker/docker/daemon/logger#Context), so `awslogs-stream-template="{{ .ContainerID }}"` would be equivalent to the default behavior.